src/main/java/jenkins/plugins/git/maintenance/MaintenanceUI.java

```diff
@@ -0,0 +1,251 @@
+package jenkins.plugins.git.maintenance;
+
+import antlr.ANTLRException;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import hudson.Extension;
+import hudson.model.ManagementLink;
+import hudson.model.PeriodicWork;
+import hudson.security.Permission;
+import hudson.util.FormValidation;
+import jenkins.model.GlobalConfiguration;
+import jenkins.model.Jenkins;
+import jenkins.plugins.git.maintenance.Logs.CacheRecord;
+import jenkins.plugins.git.maintenance.Logs.XmlSerialize;
+import net.sf.json.JSONArray;
+import net.sf.json.JSONObject;
+import org.kohsuke.accmod.Restricted;
+import org.kohsuke.accmod.restrictions.NoExternalUse;
+import org.kohsuke.stapler.QueryParameter;
+import org.kohsuke.stapler.StaplerRequest;
+import org.kohsuke.stapler.StaplerResponse;
+import org.kohsuke.stapler.bind.JavaScriptMethod;
+import org.kohsuke.stapler.interceptor.RequirePOST;
+import org.kohsuke.stapler.verb.POST;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+@Extension
+public class MaintenanceUI extends ManagementLink {
+
+    JSONObject notification = new JSONObject();
+    String OK = "OK";
+    String ERROR = "ERROR";
+
+    private static final Logger LOGGER = Logger.getLogger(MaintenanceUI.class.getName());
+
+    @Override
+    public String getIconFileName() {
+        return jenkins.model.Jenkins.RESOURCE_PATH + "/plugin/git/icons/git-maintenance.svg";
+    }
+
+    @Override
+    public String getDisplayName() {
+        return "Git Maintenance";
+    }
+
+    @Override
+    public String getUrlName() {
+        return "maintenance";
+    }
+
+    @Override
+    public String getDescription() {
+        return "Maintain your Repositories to improve git command performance.";
```

Comment:
Lower case in the middle of a sentence.
```suggestion
        return "Maintain your repositories to improve git command performance.";
```